.circleci/docker/ubuntu-cuda/Dockerfile

```
@@ -78,6 +78,11 @@ ADD ./common/install_jni.sh install_jni.sh
 ADD ./java/jni.h jni.h
 RUN bash ./install_jni.sh && rm install_jni.sh
 
+# Install NCCL for *-xenial-cuda10.1*-
+ADD ./common/install_nccl.sh install_nccl.sh
+RUN if [ "$BUILD_ENVIRONMENT" == *-xenial-cuda10.1-* ]; then bash ./install_nccl.sh; fi
+RUN rm install_nccl.sh
```

you can do
```suggestion
ARG NVCC
ADD ./common/install_nccl.sh install_nccl.sh
RUN if [ "$BUILD_ENVIRONMENT" == *-xenial-cuda10.1-* ]; then bash ./install_nccl.sh; fi
RUN rm install_nccl.sh
```
and put the NVCC flag on the build.sh file

---

`RUN if [ -n "${NVCC}" ]; bash ./install_nccl.sh; fi` -> `RUN if [ -n "${NVCC}" ]; then bash ./install_nccl.sh; fi`

---

I'm going to use the CUDA_VERSION env var instead.